precompile/contracts/warp/contract.go

```diff
@@ -28,9 +27,9 @@ const (
 	// Note: using trie write for the gas cost results in a conservative overestimate since the message is stored in a
 	// flat database that can be cleaned up after a period of time instead of the EVM trie.
 
-	SendWarpMessageGasCost uint64 = params.LogGas + 3*params.LogTopicGas + AddWarpMessageGasCost + contract.WriteGasCostPerSlot
+	SendWarpMessageGasCost uint64 = contract.LogGas + 3*contract.LogTopicGas + AddWarpMessageGasCost + contract.WriteGasCostPerSlot
```

Comment:
good catch. Warp was under `x` package but once we move it to `precompile` package we should be using `contract` package to prevent a import cycle. I will change it in https://github.com/ava-labs/subnet-evm/pull/1034